src/screens/DiscoveryScreenV2/fragment/TopicFragment.js

```diff
@@ -185,23 +185,20 @@ const TopicFragment = ({
       <>
         {renderRecentSearch(index)}
         <View style={styles.domainContainer}>
-          {(route.name === 'Followings' && item.user_id_follower !== null) ||
-          route.name !== 'Followings' ? (
-            <DomainList
-              handleSetFollow={() => handleFollow(from, true, item, index)}
-              handleSetUnFollow={() => handleFollow(from, false, item, index)}
-              key={`followedTopic-${index}`}
-              isCommunity={true}
-              onPressBody={() => __handleOnTopicPress(item)}
-              item={{
-                name: item.name,
-                image: item.icon_path,
-                isunfollowed: !item.following,
-                description: null
-              }}
-              DefaultImage={TopicsProfilePictureEmptyState}
-            />
-          ) : null}
+          <DomainList
+            handleSetFollow={() => handleFollow(from, true, item, index)}
+            handleSetUnFollow={() => handleFollow(from, false, item, index)}
+            key={`followedTopic-${index}`}
+            isCommunity={true}
+            onPressBody={() => __handleOnTopicPress(item)}
+            item={{
+              name: item.name,
+              image: item.icon_path,
+              isunfollowed: !item.following,
+              description: null
+            }}
+            DefaultImage={TopicsProfilePictureEmptyState}
+          />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering of the `DomainList` component based on the route name and `item.user_id_follower` has been removed. This means that the `DomainList` component will now always be rendered, which might not be the intended behavior. If the previous condition was necessary for certain scenarios, this change could introduce bugs or undesired effects.

```diff
-          {(route.name === 'Followings' && item.user_id_follower !== null) ||
-          route.name !== 'Followings' ? (
            <DomainList
              handleSetFollow={() => handleFollow(from, true, item, index)}
              handleSetUnFollow={() => handleFollow(from, false, item, index)}
              key={`followedTopic-${index}`}
              isCommunity={true}
              onPressBody={() => __handleOnTopicPress(item)}
              item={{
                name: item.name,
                image: item.icon_path,
                isunfollowed: !item.following,
                description: null
              }}
              DefaultImage={TopicsProfilePictureEmptyState}
            />
-          ) : null}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->